pom.xml

```diff
@@ -47,14 +47,14 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.17.0</flink.version>
+        <flink.version>1.16.2</flink.version>
```

Comment:
I am wondering why the Flink version is going down here?

Reply:
the minimum supported version should be here
just a general rule for connector AFAIK

Reply:
exactly, as this should be the minimum version we are setting that minimum version.
this was commented with @MartijnVisser offline 

I'm still not convinced about this, as I would prefer to have last version to see deprecations and start working on it and the compatibility with other versions is already tested with CI
but I'm following the rules